client/a8c-for-agencies/sections/referrals/referrals-list/style.scss

```diff
@@ -13,9 +13,3 @@
 
 	}
 }
-
```

Comment:
@cleacos I've reverted these changes. I'd like to identify if we need any hotfix and do it this way https://github.com/Automattic/wp-calypso/pull/93503/files#r1732408533 otherwise, it's going to be difficult to update dataviews in the future.

I've identified one issue with dataviews ([sort direction in view config](https://github.com/WordPress/gutenberg/issues/64816)) that I've fixed at https://github.com/WordPress/gutenberg/pull/64817 It's going to be part of a future dataviews release so we'd probably need to add a hotfix in the meantime.

Reply:
> I'm looking now at a hotfix for the sort direction.

Added one for Referrals https://github.com/Automattic/wp-calypso/pull/93503/commits/3f8535129459ea32f05dff912210c3e7b7dd2ea8, Team https://github.com/Automattic/wp-calypso/pull/93503/commits/eae923a2e439f102df4b55214ac52060fa0782af, and Sites https://github.com/Automattic/wp-calypso/pull/93503/commits/2cfdd5466255e1d7478c72e9adf2e2bbef3aaf7a

Reply:
Clemente was kind enough to walk me through a few things, and mentioned an extra need for hiding the whole view config was due to not having pagination available in a few places. I've taken a look at them and pushed a solution to do client-side pagination using `filterSortAndPaginate` (a dataviews utility):

- Team https://github.com/Automattic/wp-calypso/pull/93503/commits/4f7161e37a640875a1c8b8442ad9851fb29aa75b
- Referrals overview https://github.com/Automattic/wp-calypso/pull/93503/commits/ae3076cdbfb58fccc45b06239501f8fd2392d334
- Referrals details https://github.com/Automattic/wp-calypso/pull/93503/commits/ffc356fd3ae67bd7722a1840c97a2d6667460663

I think this should solve all the concerns and we don't need any other hotfix/extra thing.

Reply:
`filterSortAndPaginate` TIL

Cool! For the current cases, the client-side pagination works well.